Formula/saxon.rb

```diff
@@ -1,9 +1,9 @@
 class Saxon < Formula
   desc "XSLT and XQuery processor"
   homepage "https://saxon.sourceforge.io"
```

Comment:
this homepage should be also updated.

Reply:
I mentioned some options [in my first post](https://github.com/Homebrew/homebrew-core/pull/120597#issuecomment-1382823968) above:

> I think the `homepage` should also be updated from https://saxon.sourceforge.io/ to https://github.com/Saxonica/Saxon-HE/tree/main/12 - the homepage of the SaxonHE edition of Saxon. Alternatively, it could be the official homepage of Saxon, https://www.saxonica.com/, or the page on saxonica.com that links to GitHub, https://www.saxonica.com/download/java.xml. I'm open to any of these and would welcome advice from Homebrew users/maintainers about which one would be the most appropriate from a Homebrew perspective.

Reply:
yeah, once github releases are in, we can have a default github livecheck strategy in place, that might just do the job.

Reply:
Sounds good. In the meantime, I'll update the PR to switch `homepage` from sourceforge to the new GitHub repo's landing page, https://github.com/Saxonica/Saxon-HE. I think that's the best choice for `homepage`.